lib/livebook/notebook/explore/vm_introspection.livemd

```diff
@@ -0,0 +1,279 @@
+# Fun with VM introspection
+
+## Introduction
+
+In this notebook we manually establish connection to a running node,
+and then we try to retrieve and plot some interesting information
+about the system.
+
+## Setup
+
+We are definitely gonna plot some data in this notebook,
+so let's add `:vega_lite` and `:kino` for that.
+
+```elixir
+Mix.install([
+  {:vega_lite, "~> 0.1.0"},
+  {:kino, "~> 0.1.0"}
+])
+```
+
+```elixir
+alias VegaLite, as: Vl
+```
+
+## Connecting to a remote node
+
+This time we will connect to a running node manually instead of
+injecting Livebook there, because we may not want to run `Mix.install/1`
+inside it. This will also give you a better idea of how you can
+inspect a remote node in general.
+
+The first thing we need is an actually running node,
+if you don't have one at hand, you can start one like this:
+
+`iex --sname cat --cookie meow`
+
+Alternatively you can open up an empty notebook or use
+a production node run from an Elixir release.
+In any case, you can find the node name and cookie like this:
```

Comment:
```suggestion
The first thing we need is a separate Elixir node. For this example,
you can do so by opening up a new notebook, since Livebook
automatically starts each notebook as a remote node.

In practice, you may want to start an existing Elixir system,
such as by running the following in your production app:

    elixir --sname my_app -S mix run

Or by connecting to a production assembled via
[`mix release`](https://hexdocs.pm/mix/Mix.Tasks.Release.html).

Once you start a new notebook, you can find the node name
and cookie by running this:
```